skribenten-backend/src/main/resources/application.conf

```diff
@@ -87,5 +87,14 @@ skribenten {
         clientSecret = ${AZURE_APP_CLIENT_SECRET}
         preAuthApps = ${AZURE_APP_PRE_AUTHORIZED_APPS}
     }
+    valkey {
```

Comment:
Jeg tror kanskje jeg ville unngått å ha optional substitution her, da får vi bedre feilmeldinger om verdiene mangler. For å få det til må du eksplisitt sette noen falske verdier i application-local.conf for disse feltene.

Reply:
Ja, er jo ein tradeoff her, men trur eg er einig.